### ISSUE TITLE
On the training of pytorch model

### ISSUE THREAD

**Issue by kingsleyluoxin** [NVlabs/PWC-Net#20]
What accuracy can be obtained if using Pytorch codes to train the model? Do you have the training code in Python version? Thank you so much~

**Comment by luvegood** [comment]
We ported the trained weight from Caffe to PyTorch, thus you may refer to the training accuracies in the paper for detail: https://arxiv.org/pdf/1709.02371.pdf, such as table 1 & 3

For training code, you may refer to another github repository https://github.com/ClementPinard/FlowNetPytorch, though you need to make changes according to the detailed training method in the pwc-net paper.

**Comment by kingsleyluoxin** [comment]
Thanks so much for you detailed reply!

**Comment by xianshunw** [comment]
@kingsleyluoxin did you train the pytorch vision code? How is the result？ I tried to train the pytorch version of the code, the result is very bad. I don't know why.
